### DISCUSSION TITLE
ERR: dial tcp: lookup rtsp on 192.168.65.5:53: no such host

### DISCUSSION THREAD

**Discussion by plewyllie** [bluenviron/mediamtx#519] [Questions]
Hi there,

Thanks for all your efforts!

I'm trying to setup a simple rtsp proxy to expose my ip camera via a docker container. 

With this command:
`docker run --rm -it --network=host -e RTSP_PATHS_TEST_SOURCE=rtsp://admin:password@192.168.126.208/h264/ aler9/rtsp-simple-server`

Log output:
2021/08/18 18:20:06 I [0/0] rtsp-simple-server v0.17.2
2021/08/18 18:20:06 I [0/0] [path test] created
2021/08/18 18:20:06 I [0/0] [path test] [rtsp source] started
2021/08/18 18:20:06 I [0/0] [RTSP] UDP/RTP listener opened on :8000
2021/08/18 18:20:06 I [0/0] [RTSP] UDP/RTCP listener opened on :8001
2021/08/18 18:20:06 I [0/0] [RTSP] TCP listener opened on :8554
2021/08/18 18:20:06 I [0/0] [RTMP] listener opened on :1935
2021/08/18 18:20:06 I [0/0] [HLS] listener opened on :8888
2021/08/18 18:20:06 I [0/0] [path test] [rtsp source] ready
2021/08/18 18:20:09 I [0/0] [path test] [rtsp source] ERR: dial tcp: lookup rtsp on 192.168.65.5:53: no such host

It seems to connect (I can see RTP traffic flowing on Wireshark), however, it then sends a TEARDOWN (TEARDOWN rtsp://rtsp://192.168.126.208/h264/ RTSP/1.0\r\n) not sure if this is causing a failure but it's prepending rtsp:// twice to the URL. I can see it tries to do an A query for "rtsp" to the DNS server, which does not have an answer for this.

FYI, I can connect to this stream using my VLC client at the same URL.

Cheers
Pieter

**Comment by aler9** [comment]
Hello, this is a communication bug between the server and the specific model of your camera, please open an issue and provide a network dump by following the instructions in the issue template.

[a comment by an automated account is not shown]
